core/liblwm2m.h

```diff
@@ -372,7 +373,8 @@ typedef enum
 typedef struct _lwm2m_server_
 {
     struct _lwm2m_server_ * next;   // matches lwm2m_list_t::next
-    uint16_t          shortID;      // matches lwm2m_list_t::id
+    uint16_t          secObjInstID; // matches lwm2m_list_t::id
```

Comment:
With the secObjInstId it's very easy to access the server short ID. So even if required, it would be easy.